README.md

```diff
@@ -58,7 +58,7 @@ elasticsearch_exporter --help
 | es.indices_mappings     | 1.2.0                 | If true, query stats for mappings of all indices of the cluster.                                                                                                                                                                                                                                                                                                                      | false |
 | es.aliases              | 1.0.4rc1              | If true, include informational aliases metrics.                                                                                                                                                                                                                                                                                                                                       | true |
 | es.shards               | 1.0.3rc1              | If true, query stats for all indices in the cluster, including shard-level stats (implies `es.indices=true`).                                                                                                                                                                                                                                                                         | false |
-| es.snapshots            | 1.0.4rc1              | If true, query stats for the cluster snapshots.                                                                                                                                                                                                                                                                                                                                       | false |
+| collector.snapshot      | 1.0.4rc1              | If true, query stats for the cluster snapshots. Renamed to collector.snapshots since version 1.7.0                                                                                                                                                                                                                                                                                    | false |
```

Comment:
I don't think renamed to collector.snapshot makes sense on the collector.snapshot entry. Maybe this should be `Renamed from es.snapshots in version 1.7.0`

Reply:
I agree. Thx